modules/ROOT/pages/dataweave-selectors.adoc

```diff
@@ -399,28 +467,33 @@ payload..
 
 === Example
 
-Shows how to select only descendants names
+This example selects the descending `name` values.
 
 ==== Input
 
+The input payload contains a set of `name` elements that are nested
+at different levels.
+
 [source,xml, linenums]
 ----
-
 <users>
+  <user>
+    <name>Weave</name>
     <user>
-        <name>Weave</name>
-        <user>
-            <name>BAT</name>
-            <user>
-                <name>BDD</name>
-            </user>
-        </user>
+      <name>BAT</name>
+      <user>
+        <name>BDD</name>
+      </user>
     </user>
+  </user>
 </users>
 ----
 
 ==== Source
 
+The DataWeave script uses `..` to select the values of all `name` elements
+from the input payload and output those values into a JSON array.
+
```

Comment:
Note that `payload..name` (the script below) returns the same output as `payload..*name`.